tree/dataframe/src/RLoopManager.cxx

```diff
@@ -822,7 +822,8 @@ void RLoopManager::EvalChildrenCounts()
 
 /// Start the event loop with a different mechanism depending on IMT/no IMT, data source/no data source.
 /// Also perform a few setup and clean-up operations (jit actions if necessary, clear booked actions after the loop...).
-void RLoopManager::Run()
+/// The jitting phase is skipped if the `jit` parameter is `false` (unsafe, use with care).
```

Comment:
why would one pass jit False here?

Reply:
the rest of the commit introduces the one and only usage of this flag